website/src/pages/index.js

```diff
@@ -1,92 +1,78 @@
-      <>
-        Indexed data can be quickly joined across disparate datasets and aggregated at different levels of precision.
-      </>
-    ),
-  },
-  {
-    title: 'Algorithms and optimizations',
-    imageUrl: 'images/icon-chart.svg',
-    description: (
-      <>
-        H3 enables a range of algorithms and optimizations based on the grid, including nearest neighbors, shortest path, gradient smoothing, and more.
-      </>
-    ),
-  },
-];
+const TextContainer = styled.div`
+  max-width: 800px;
+  padding: 64px 112px;
+  width: 70%;
+  font-size: 14px;
 
-function Feature({imageUrl, title, description}) {
-  const imgUrl = useBaseUrl(imageUrl);
-  return (
-    <div className={clsx('col col--4', styles.feature)}>
-      {imgUrl && (
-        <div className="text--center">
-          <img className={styles.featureImage} src={imgUrl} alt={title} />
-        </div>
-      )}
-      <h3>{title}</h3>
-      <p>{description}</p>
-    </div>
-  );
-}
+  h2 {
+    font: bold 32px/48px;
+    margin: 24px 0 16px;
+    position: relative;
+  }
+  h3 {
+    font: bold 16px/24px;
+    margin: 16px 0 0;
+    position: relative;
+  }
+  h3 > img {
+    position: absolute;
+    top: -4px;
+    width: 36px;
+    left: -48px;
+  }
+  hr {
+    border: none;
+    background: #e1e8f0;
+    height: 1px;
+    margin: 24px 0 0;
+    width: 32px;
+    height: 2px;
+  }
+  @media screen and (max-width: 768px) {
+    max-width: 100%;
+    width: 100%;
+    padding: 48px 48px 48px 80px;
+  }
+`;
 
-export default function Home() {
-  const context = useDocusaurusContext();
-  const {siteConfig = {}} = context;
+export default function IndexPage() {
   return (
-    <Layout
-      title={`${siteConfig.title}`}
-      description="Hexagonal hierarchical geospatial indexing system">
-      <header className={clsx('hero hero--primary', styles.heroBanner)}>
-        <div className="container">
-          <h1 className="hero__title">{siteConfig.title}</h1>
-          <p className="hero__subtitle">{siteConfig.tagline}</p>
-          <div className={styles.buttons}>
-            <Link
-              className={clsx(
-                'button button--outline button--secondary button--lg',
-                styles.getStarted,
-              )}
-              to={useBaseUrl('docs/')}>
-              Get Started
-            </Link>
-          </div>
-        </div>
-      </header>
-      <main>
-        {features && features.length > 0 && (
-          <section className={styles.features}>
-            <div className="container">
-              <div className="row">
-                {features.map((props, idx) => (
-                  <Feature key={idx} {...props} />
-                ))}
-              </div>
-            </div>
-          </section>
-        )}
-      </main>
+    <Layout title="Home" description="H3">
+      <BrowserOnly>
+        {() => {
+          const HomeExplorer = require("../components").HomeExplorer;
+          return <HomeExplorer />;
```

Comment:
For my info, why is the dynamic import needed here? Do the 3rd-party libs access browser properties on import?

Reply:
My mistake, the dynamic import does not seem to avtually be needed.

Reply:
Actually DocuSaurus server side rendering will crash without it.